### ISSUE TITLE
Daemon wont start

### ISSUE THREAD

**Issue by buxxi** [DavidRisch/steamvr_utils#4]
Hi, saw a thread on /r/linux_gaming about this project so I decided to try it out so I wont have to write the script myself, but I can't get it to launch.

Getting the following output
```
[~/steamvr_utils] ./scripts/steamvr_utils.py daemon
2021-01-17 19:53:33 [DEBUG]: dry_run: False
2021-01-17 19:53:33 [DEBUG]: 'pactl list short sinks':
1	alsa_output.pci-0000_03_00.1.hdmi-stereo-extra2	module-alsa-card.c	s16le 2ch 44100Hz	SUSPENDED
2	alsa_output.pci-0000_00_1f.3.analog-stereo	module-alsa-card.c	s16le 2ch 44100Hz	IDLE

2021-01-17 19:53:33 [DEBUG]: normal sink: alsa_output.pci-0000_00_1f.3.analog-stereo
2021-01-17 19:53:33 [DEBUG]: vr sink: alsa_output.pci-0000_03_00.1.hdmi-stereo-extra2
2021-01-17 19:53:33 [INFO ]: SteamvrUtils stating daemon:
Terminated
```

The only thing I changed in the config was disable the base station feature and the regex for my headphones:
```
[~/steamvr_utils/config] diff config_template.yaml config.yaml 
9c9
<   enabled: true  # Boolean. Enable the Base Station component.
---
>   enabled: false  # Boolean. Enable the Base Station component.
16c16
<   normal_sink_regex: ''  # Regex. Used to find the audio sink of the regular audio device. Leave empty to detect automatically (not recommended).
---
>   normal_sink_regex: '.*analog.*'  # Regex. Used to find the audio sink of the regular audio device. Leave empty to detect automatically (not recommended).
```

It also guesses wrong on the "vr sink" which should be hdmi-stereo-extra3, but the "device.product.name" wont show up in "pacmd list-cards" for my Index before starting SteamVR.

**Comment by DavidRisch** [comment]
> the "device.product.name" wont show [...] before starting SteamVR

The port does not show up when SteamVr is not running, that is normal.

I think I fixed the bug you ran into in b5ab021d. Please try the new version.

**Comment by buxxi** [comment]
Yeah, now it starts and it changes the audio output for the vrtours when starting steamvr, thanks!

Just a thought:
- It probably shouldn't log warnings like "Failed to find any port on any card matching" on startup if it depends on steamvr being started after this script. Since that will always be the case that it can't resolve it until after vrcompositor has been started.
